projects/1.19/assets/fabric-seasons/seasons/lang/zh_cn.json

```
@@ -0,0 +1,21 @@
+{
+  "tooltip.seasons.spring": "春",
+  "tooltip.seasons.summer": "夏",
+  "tooltip.seasons.fall": "秋",
+  "tooltip.seasons.winter": "冬",
+  "tooltip.seasons.faster_grow": "在此季节生长更快",
+  "tooltip.seasons.normal_grow": "在此季节正常生长",
+  "tooltip.seasons.slowed_grow": "在此季节生长更慢",
+  "tooltip.seasons.not_grow": "在此季节不会生长",
+  "tooltip.seasons.show_more": "按%s显示更多信息",
+
+  "chat.seasons.mod_installed": "看来你已经安装了%s！",
+  "chat.seasons.compatibility": "安装拓展可完全兼容Fabric Seasons。",
```

```suggestion
  "chat.seasons.compatibility": "安装拓展可解锁Fabric Seasons的完整兼容特性。",
```
属于意译（），可以看下加了addon后会不会有新的兼容

---

大概是这个意思，这里的拓展mod应该是指[这个](https://www.mcmod.cn/class/10435.html)